test/cpp/naming/BUILD

```diff
@@ -34,6 +34,14 @@ grpc_py_binary(
     ],
 )
 
+grpc_py_binary(
+    name = "resolver_component_tests_runner_no_srv",
```

Comment:
Do we need a whole new test harness for this?  Wouldn't it be less invasive to just run individual tests with different channel args set?